classes/journal/JournalDAO.inc.php

```diff
@@ -204,7 +204,7 @@ function deleteJournalById($journalId) {
 	 * @param $enabledOnly boolean True iff only enabled jourals wanted
 	 * @param $rangeInfo object optional
 	 * @param $sortBy JOURNAL_FIELD_... optional sorting parameter
-	 * @param $searchField JOURNAL_FIELD_... optional filter parameter
+	 * @param $searchField JOURNAL_FIELD_... optional filter parameter; DEPRECATED - use getBySetting() instead
```

Comment:
I'd rather not deprecate function parameters; if it happens one day that we remove it, the risks of hard-to-debug parameter list mismatches are pretty high. If it's that a specific `$searchField` is not supported here, best to specifically assert that it's not used, I think.